sdk/include/psdk/WbemTime.h

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
We use lowercase names for these include files (because ROS can be built on case-sensitive OSes).
Also, as Mark points out, this file must look MUCH less like Microsoft's version in order to be acceptable. See chstring.h for an example.

Reply:
Thanks for review.

I'll fix now.

Reply:
And add one space before the 'star' character (code style).